packages/shared-core/src/filters.ts

```diff
@@ -323,6 +323,35 @@ export const buildQuery = (filter: SearchFilter[]) => {
   return query
 }
 
+// The frontend can send single values for array fields sometimes, so to handle
+// this we convert them to arrays at the controller level so that nothing below
+// this has to worry about the non-array values.
+export function fixupFilterArrays(filters: SearchFilters) {
+  const arrayFields = [
```

Comment:
This could come from a type/array next to the SearchFilterOperator. So when we add a new operator, we don't need to look for all the possible usages, being centralised in a single place

Reply:
Yeah, I was thinking about this. I really dislike having this list here, because as you mentioned, it's so detached from the type that it would be easy to miss upon adding a new field. Do you have some preferred way of signaling it in the type? Or would you recommend just putting this definition near the type?

Reply:
Tried a lil' something I think you might like. Let me know!